<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>715</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>105</red>
        <green>84</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>105</red>
        <green>84</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>105</red>
        <green>84</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>105</red>
        <green>84</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Select Character Filters</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="primaryArchetypes">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Primary Archetype</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="secondaryArchetypes">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Secondary Archetype</string>
     </property>
    </item>
   </widget>
   <widget class="QScrollArea" name="archetypeDescSelect">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>160</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>160</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>160</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>160</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>160</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>160</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>160</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>160</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>160</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 160, 160);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>159</width>
       <height>109</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Unselect Archetypes</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="archetypeDesc">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startProgram">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>580</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 16pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(28, 113, 2);
border: 2px solid #555;
border-radius: 20px;
border-style: outset;</string>
    </property>
    <property name="text">
     <string>Start Program</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/OptimisationLogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>571</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="abilities">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="leftColumn">
       <item>
        <layout class="QHBoxLayout" name="strLayout">
         <item>
          <widget class="QLabel" name="strName">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
           </property>
           <property name="text">
            <string>Strength</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="intLayout">
         <item>
          <widget class="QLabel" name="intName">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
           </property>
           <property name="text">
            <string>Intelligence</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="midColumn">
       <item>
        <layout class="QHBoxLayout" name="dexLayout">
         <item>
          <widget class="QLabel" name="dexName">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
           </property>
           <property name="text">
            <string>Dexterity</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>12</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="wisLayout">
         <item>
          <widget class="QLabel" name="wisName">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
           </property>
           <property name="text">
            <string>Wisdom</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightColumn">
       <item>
        <layout class="QHBoxLayout" name="conLayout">
         <item>
          <widget class="QLabel" name="conName">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
           </property>
           <property name="text">
            <string>Constitution</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="chaLayout">
         <item>
          <widget class="QLabel" name="chaName">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
           </property>
           <property name="text">
            <string>Charisma</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>25</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>480</y>
      <width>561</width>
      <height>67</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="skills">
     <item>
      <layout class="QVBoxLayout" name="skillColumn1">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="skillColumn2"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="skillColumn3"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="skillColumn4"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="skillColumn5"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="skillColumn6"/>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>641</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 160, 160);</string>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="renderHints">
     <set>QPainter::TextAntialiasing</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
    </property>
    <property name="text">
     <string>Abilities</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>460</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
    </property>
    <property name="text">
     <string>Skills</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
    </property>
    <property name="text">
     <string>Archetypes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>47</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="pixmap">
     <pixmap>../Images/OptionalText.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>641</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 160, 160);</string>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="renderHints">
     <set>QPainter::TextAntialiasing</set>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>675</y>
      <width>401</width>
      <height>50</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 15, 47);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>640</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="filtersLayout"/>
   </widget>
   <widget class="QPushButton" name="mainMenuBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(172, 3, 3);
border: 2px solid #555;
border-radius: 10px;
border-style: outset;
font: 75 10pt &quot;Rockwell Extra Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Return to 
 Main Menu</string>
    </property>
   </widget>
   <zorder>graphicsView_2</zorder>
   <zorder>graphicsView</zorder>
   <zorder>primaryArchetypes</zorder>
   <zorder>secondaryArchetypes</zorder>
   <zorder>archetypeDescSelect</zorder>
   <zorder>archetypeDesc</zorder>
   <zorder>startProgram</zorder>
   <zorder>label</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>error</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>mainMenuBtn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
